.bazelrc

```diff
@@ -53,6 +53,7 @@
 #     nogcp:        Disable GCS support.
 #     nohdfs:       Disable hadoop hdfs support.
 #     nonccl:       Disable nccl support.
+#     gles:         Build with OpenGL ES support (TF Lite only).
```

Comment:
I'm not fan of adding a flag at the global level.  However, read on to my next comment.

Reply:
Basically, the only thing that this flag does is add `-lEGL -lGLESv2` to shared libraries. I am okay with dropping the config target, provided I can add these flags for linux targets. I am not familiar with bazel, though, maybe there's a way I can add them without adding a config flag?

Reply:
Sorry, missed this comment of yours.

Yes, you can easily add --linkopt -lEGL --linkopt -lGLESv2 when running bazel in command line, or you can add linkopts = ["-lEGL", "-lGLESv2"] in your BUILD file.  If you grep for lEGL or lGLES in BUILD files, you might be able to find examples.  You can just add those to your terminal cc_library or cc_binary that employs the GPU delegate on your end.

Reply:
So if I understand correctly, the only way right now for non-mobile targets to use the GPU delegate is to have a binary built alongside the tensorflow source code? I guess this makes sense, with a lack of an easy way to get a "packaged" library with headers.

Okay, guess this makes sense, though loading the delegate from Python will probably be somewhat complicated (then again, the API is marked experimental for a reason, and the client code may just preload the required libraries).